docker/README.md

```diff
@@ -0,0 +1,50 @@
+# How to use docker to spin up WebSiteOne
+
+## Prerequisites
+
+In order to run this container you'll need docker installation
+* [Windows](https://docs.docker.com/docker-for-windows/)
```

Comment:
So the seed_databash.sh script won't work right now inside of a windows docker machine because of line endings being copied over from windows git:

/usr/bin/env: ‘bash\r’: No such file or directory

Perhaps we should note this explicitly, and open a ticket to do windows support? 
@brotich